mantle/kola/tests/coretest/core.go

```diff
@@ -92,6 +93,29 @@ func init() {
 	})
 }
 
+func TestStableRoot() error {
+	unit := "coreos-root-symlink.service"
+	link := "/dev/disk/by-label/coreos-root-disk"
+
+	if err := checkService(unit); err != nil {
+		return err
+	}
+
+	c := exec.Command("journalctl", "-o", "cat", "-u", unit)
+	out, err := c.Output()
+	if err != nil {
+		return fmt.Errorf("journalctl: %s", err)
+	}
+	if !strings.Contains(string(out), link) {
```

Comment:
It'd be better to write an Ignition config that uses the symlink to modify the partition table, perhaps just by resizing the root partition.  That would confirm not only that we created the symlink, but that we did it before ignition-disks ran, and that Ignition was able to use it successfully.